mods/ra/weapons/ballistics.yaml

```diff
@@ -49,96 +26,57 @@
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
-105mm:
-	ReloadDelay: 70
-	Range: 4c768
-	Report: cannon1.aud
-	Burst: 2
-	BurstDelay: 4
+25mm:
+	Inherits: ^Cannon
+	ReloadDelay: 13
+	Range: 4c0
+	Report: cannon2.aud
 	Projectile: Bullet
-		Speed: 682
-		Image: 120MM
+		Speed: 853
+		Image: 50CAL
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
-		Damage: 40
+		Damage: 16
 		Versus:
-			None: 20
-			Wood: 75
-			Light: 75
-			Concrete: 50
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
-	Warhead@3EffGround: CreateEffect
+			None: 30
+			Wood: 40
+			Light: 100
+			Heavy: 40
+			Concrete: 30
+	-Warhead@2Smu: LeaveSmudge
+	Warhead@3Eff: CreateEffect
 		Explosions: small_explosion
-		ImpactSounds: kaboom12.aud
 		InvalidImpactTypes: Water
 	Warhead@4EffWater: CreateEffect
 		Explosions: small_splash
-		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
+90mm:
+	Inherits: ^Cannon
+
+105mm:
+	Inherits: ^Cannon
+	ReloadDelay: 70
+	Burst: 2
+	BurstDelay: 4
+
 120mm:
```

Comment:
You forgot the `Inherits` here!  This quite effectively nerfs the mammoth tank :smile: